### ISSUE TITLE
How was your overfit level?

### ISSUE THREAD

**Issue by adarc8** [pfriedri/wdm-3d#13]
When I trained on large scale MRI dataset of around 25k samples, I experince no overfit (3+ days training).
However when using BraTS, which is much smaler dataset, overfit starts at arond epoch 1000 (2.5 days training).
When you train on brats, did you guys experience overfit like I do?

attaching examples

brats overfit:
![Image](https://github.com/user-attachments/assets/6e3e2456-c078-47a5-96d3-9406a22ceee5)

25k dataset no overfit:

![Image](https://github.com/user-attachments/assets/fa9a2ecd-4683-4738-8406-800876ec1f1f)

**Comment by pfriedri** [comment]
@adarc8 I don't really get the question. What do you mean with overfitting in this case? Why do you have a validation set in an unconditional generation task and what would be your "ground truth"?

**Comment by adarc8** [comment]
If the task was about taking a pure noise and generate image via 1 step, I would agree that validation set is not needed, but in diffusion model, every training iteration has a noisy input which is strongly conditioned on the original clean image, as your loss component is (predict_clean, gt_clean).

for ur quesiton, the "GT" is the the clean original image

**Comment by pfriedri** [comment]
@adarc8 I don't understand your question. When you do unconditional image generation, there is just a "training set". I don't see how you want to compute any validation metrics, as the goal of the model is to generate unseen images. So what do you compare with what?
